src/integration-tests/model-in-image/mii-sample-wrapper/generate-sample-doc.sh

```diff
@@ -157,10 +171,10 @@ for phase in initial update1 update2 update3; do
    
   # setup script for the configmap
 
-  if [ "$configmap" = "true" ]; then
+  if [ "$configmap" != "None" ]; then
```

Comment:
Purpose of this is to leave behind a utility shell script for creating the configmap - but it looks like update4 might overwrite the utility script generated by a previous update.   I could be wrong...

Reply:
generate-sample-doc.sh generates a separate *config.sh" for each update use case.